src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -249,6 +251,15 @@ public void onTraversalStart() {
             }
         }
 
+        //Allele length and Genotype length annotations need to be subset or removed if alleles are dropped so we need to parse the header for annotation count types
+        for(final VCFFormatHeaderLine formatHeaderLine : inputHeader.getFormatHeaderLines()) {
```

Comment:
Occasionally the header is wrong, but it's usually A should be R or vice versa, not const should be A.  I think this is fine since we're just using this as a list to remove, but it's something to keep in mind.

Reply:
Hmm, in that case I'll add a warning for the logs that at least lists which annotations have been removed for easier debugging in the future.